samtools.wdl

```diff
@@ -383,3 +395,37 @@ task View {
                       category: "advanced"}
     }
 }
+
+task FilterShortReadsBam {
+    input {
+        File bamFile
+        String outputPathBam
+        String outputPathBamIndex
```

Comment:
This should be based on the `outputPathBamIndex`, it shouldn't be an input.

Reply:
You mean outputPathBam?

Reply:
oh yes, sorry, should be based on `outputPathBam`